src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -189,20 +193,13 @@ protected virtual void Initialize()
                     File.WriteAllText(hostConfigFilePath, "{}");
                 }
 
-                if (ScriptConfig.HostConfig.IsDevelopment)
+                if (ScriptConfig.HostConfig.IsDevelopment || InDebugMode)
                 {
+                    // If we're in debug/development mode, use optimal debug settings
                     ScriptConfig.HostConfig.UseDevelopmentSettings();
                 }
-                else
-                {
-                    // TEMP: Until https://github.com/Azure/azure-webjobs-sdk-script/issues/100 is addressed
```

Comment:
Addressing this issue that we lost track of - it came up recently in a PR. When debugging or testing in portal (InDebugMode == true) we want to run with debug settings, but in production steady state, we want to run with production defaults.